src/shared/install.c

```diff
@@ -68,6 +68,16 @@ typedef struct {
 
 static int unit_file_lookup_state(UnitFileScope scope, const LookupPaths *paths, const char *name, UnitFileState *ret);
 
+bool unit_type_no_alias(UnitType type) {
+        return IN_SET(type,
+                      UNIT_BUSNAME,
+                      UNIT_MOUNT,
+                      UNIT_AUTOMOUNT,
+                      UNIT_SWAP,
+                      UNIT_SLICE,
+                      UNIT_SCOPE);
+}
```

Comment:
Well, if we move this here, then maybe we should invert the thing, as there's no benefit in having this as "negative" bool. I.e. call this "unit_type_may_alias()" or so...